src/rockstor/system/ssh.py

```diff
@@ -151,10 +152,14 @@ def rsync_for_sftp(chroot_loc):
 
     copy("/bin/bash", "{}/bin".format(chroot_loc))
     copy("/usr/bin/rsync", "{}/usr/bin".format(chroot_loc))
+    
+    ld_linux_so = "/lib64/ld-linux-x86-64.so.2"
+    if (platform.machine() == "aarch64"):
+	ld_linux_so = "/lib64/ld-linux-aarch64.so.1"
```

Comment:
You've got a tab here instead of four spaces:
```suggestion
    ld_linux_so = "/lib64/ld-linux-aarch64.so.1"
```